packages/valory/skills/liquidity_rebalancing_abci/test_tools/integration.py

```diff
@@ -49,10 +49,15 @@ class AMMIntegrationBaseCase(
 ):
     """Base test class for integration tests in a Hardhat environment, with AMM interaction."""
 
+    @classmethod
+    def _setup_class(cls, **setup_class_kwargs: Any) -> None:
+        """Continue setting up the class."""
```

Comment:
The abstract method had to be implemented.

Reply:
What is it overriding on the base?

Reply:
This one:
https://github.com/valory-xyz/open-autonomy/blob/d96e69bdc046a0bcb0f4abfc930036aba769bf61/plugins/aea-test-autonomy/aea_test_autonomy/docker/base.py#L260-L263